politeiad/backend/gitbe/gitbe.go

```diff
@@ -419,9 +419,16 @@ func verifyContent(metadata []backend.MetadataStream, files []backend.File, file
 // backend.File that is completely filled out.
 //
 // This function must be called with the lock held.
-func loadRecord(path, id string) ([]backend.File, error) {
+func loadRecord(path, id, version string) ([]backend.File, error) {
```

Comment:
I think this endpoint (or maybe a new one) should also return the number of versions.  It is hard to specify a version if I don't know how many actually exists.

For example, github diff today allows you to know how many version of a comment exist, so that you can chose which comment you want to diff.

Correct me if I am mistaken but, the current implementation only allows me to fetch the latest version only, since I don't know how many versions exist.

Reply:
also the latest version is the current one I have, so in the diff case, it would be different at all.

Reply:
more or less, how would I know there are 5 versions for example

Reply:
Because if you don't specify the version you will get the latest version by default. From there you can tell what are the past versions.